server/text_generation_server/models/__init__.py

```diff
@@ -1248,7 +1248,7 @@ def get_model(
             revision=revision,
             quantize=quantize,
             speculator=speculator,
-            dtype=dtype,
+            dtype=torch.bfloat16,
```

Comment:
Should be passed through with `default_dtype` I think?

Reply:
ah good catch! updating to use `default_dtype` in the latest commit